.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -12,16 +12,24 @@ assignees: ''
 <!-- learned, some awesome projects you worked on, something new in Data -->
 <!-- Science, Machine Learning, Automation, Cloud, Containers, Hardware. -->
 
-**Title of the talk**
+#### Title of the talk
 <!-- Please include a short title -->
 
-**Description**
+#### Description
 <!-- Please include brief abstract about your talk. -->
 
-**Duration (including Q&A)**
+#### Duration (including Q&A)
 <!-- Please add estimated duration of the talk including Q&A -->
 
-**Prerequisites**
+#### Prerequisites
 <!-- If you have any specific requirement from audience -->
 <!-- for the talk i.e. software, tools set up on machine etc, please -->
 <!-- include it here -->
+
+#### Checklist
+[Place an '[x]' (no spaces) in all the fields you agree to. You can keep the fields blank on which you don't agree.]
```

Comment:
This line should be comment.

Reply:
This was taken from, https://github.com/helm/charts/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Let's make it a comment